pkg/process/exec_conf.go

```diff
@@ -54,7 +54,7 @@ var (
 
 // SaveConfig will save all flags with default values to outfilewith specific
 // values specified in 'overrides' overridden.
-func SaveConfig(flagset *pflag.FlagSet, outfile string, overrides map[string]interface{}) error {
+func SaveConfig(flagset *pflag.FlagSet, outfile string, overrides map[string]interface{}, saveAllDefaults bool) error {
```

Comment:
Can we use a separate name for this, instead of boolean flag. e.g. `SaveConfigUser`, `SaveConfig`?

Reply:
Added `SaveConfigWithAllDefaults` function.